inc/feedwriter.php

```diff
@@ -6,8 +6,8 @@ class FeedWriter {
 	public function __construct($id) {
 		$this->feedID = $id;
 		$this->baseURL =
-			'http://' .
-			$_SERVER['HTTP_HOST'] .
+			http_https() .
+			$_SERVER['SERVER_NAME'] .
```

Comment:
Is SERVER_NAME always defined? What's the difference between HTTP_HOST and SERVER_NAME?